src/components/file-explorer/components/file-explorer-collapsible-sidbar/file-explorer-collapsible-sidebar.module.css

```
@@ -0,0 +1,44 @@
+.file-explorer-collapsible-sidebar {
+  background-color: hsl(360deg 100% 100% / 90%);
+  background-image:
+    url("/assets/background/background_1.svg"),
+    url("/assets/background/background_2.svg");
+  background-position:
+    right top,
+    left bottom;
+  background-repeat: no-repeat;
+  background-size: 50%, 50%;
+  color: var(--paletter-text-primary);
+  box-shadow: -40px 40px 80px -8px rgb(var(--palette-grey-500) / 24%);
+
+  backdrop-filter: blur(20px);
+
+  position: fixed;
+  inset-block-start: 0;
+  inset-inline-end: 0;
+  overflow-y: auto;
```

الکی از جزئیات استفاده نکن
```suggestion
  overflow: auto;
```